Engine/World.cs

```diff
@@ -96,6 +129,19 @@ public void RemoveEntity(Entity entity)
             _entitiesRemovedBuffer.Add(entity);
         }
 
+        /// <summary>
+        /// Updates the position of an entity from the world.
+        /// </summary>
+        /// <param name="entity">Entity to be updated</param>
+        public void UpdateEntityRegion(Entity entity)
+        {
+            if (Entities.Contains(entity))
+            {
+                RemoveEntity(entity);
+            }
+            AddEntity(entity);
+        }
+
```

Comment:
Is there a better way to do this? @Nowp

Reply:
![image](https://user-images.githubusercontent.com/13500119/144671638-2d47d44f-bac8-4249-b643-93b43fea155e.png)